CHANGES.md

```diff
@@ -1,4 +1,30 @@
-## edge-19-2.5
+## edge-19.3.1
+
+* CLI
+  * Introduce a check for NET_ADMIN in `linkerd check`
+  * Fixed permissions check for CRDs
+  * Include kubectl version check as part of `linkerd check` (thanks @yb172!)
+  * Removed the `--api-port` flag (thanks @paranoidaditya!)
+  * Added TCP stats to the stat command, under the `-o wide` and `-o json` flags
+* Proxy
+  * Increased the inbound/router cap on MAX_CONCURRENT_STREAMS
+  * The `l5d-remote-ip` header is now set on inbound requests and outbound
+    responses
+* Web UI
+  * Jobs now appear in the dashboard and Grafana (thanks, @Pothulapati!)
+  * Fixed sidebar not updating when resources were added/deleted (thanks
+    @liquidslr!)
+  * Added filter functionality to the metrics tables
+* Internal
+  * Added more log errors to the integration tests
+  * Recreate the `mutatingwebhookconfigurations` when the proxy injector is
+    restarted, so that the MWC always picks up the latest config template during
+    version upgrade
```

Comment:
I recommend moving this bullet to a "Controller" section, since it's a user-facing change and not strictly internal.